install.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+set -e
+
+VERSION="0.100.2"
```

Comment:
Is it possible to get latest available version from Github? So we would not need to change it on every release

Reply:
It is possible, but I don't think it's a good idea. By including the version in the script it is possible to lock to a certain version by referencing either the commit hash or a Git tag in this repository. It does add the overhead of performing the bump, but 1) we already have that through the VERSION field in `action.yml` and 2) we can automate that.